### PR TITLE
D8CORE-851 Fix margin for figures

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Override browser styles for `<figure>` tags

# Need Review By (Date)
- eventually

# Urgency
- low

# Steps to Test
1. checkout this branch
1. add a large image in the WYSIWYG, add a caption to it and align it to the left
1. add a blob of text below or above it
1. save and verify the image left alignment is the same as the text left alignment

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/scss/base/_typography.scss [hunk 1]
```diff
@@ -39,3 +39,7 @@
 .su-quote-text {
   @include type-b;
 }
+
+figure {
+  margin: 0;
```
Comment (sherakama) [inline src/scss/base/_typography.scss]: This is already in Decanter. (https://github.com/SU-SWS/decanter/blob/master/core/src/scss/elements/image/_figure.scss)

Why is it needed in stanford_basic?

Comment (pookmish) [reply src/scss/base/_typography.scss]: ahh why indeed.... :thinking:

Comment (sherakama) [reply src/scss/base/_typography.scss]: close?

Comment (pookmish) [reply src/scss/base/_typography.scss]: we could if we can figure out why its not added to the site right now

Comment (sherakama) [reply src/scss/base/_typography.scss]: https://github.com/SU-SWS/decanter/tree/6.0.0-alpha.1/core/src/scss/elements/image

The figure partial is missing in 6.0.0-alpha.1

Either a new release needs to be punched or just wait.

Comment (pookmish) [reply src/scss/base/_typography.scss]: works for me. it'll fix itself later

**Pull request closed without merging**